http.go

```diff
@@ -21,7 +22,12 @@ func (om *OpenMock) startHTTP() {
 		}).Info()
 	}))
 	if om.CorsEnabled {
-		e.Use(middleware.CORS())
+		e := echo.New()
```

Comment:
```suggestion
```

Reply:
`e` should be the one already created above

Reply:
@zhouzhuojie fixed